packages/apps-config/src/endpoints/productionRelayPolkadot.ts

```diff
@@ -383,7 +383,7 @@ export const prodParasPolkadot: Omit<EndpointOption, 'teleport'>[] = [
     paraId: 2086,
     providers: {
       Dwellir: 'wss://kilt-rpc.dwellir.com',
-      'KILT Protocol': 'wss://spiritnet.kilt.io/',
+      BOTLabs: 'wss://spiritnet.kilt.io/',
```

Comment:
As per the CI failure, we should just move this entry up to be alphabetical.

Reply:
Thanks for the quick review and sorry for the sloppy mistakes. Should be fixed: [32356ea](https://github.com/polkadot-js/apps/pull/10032/commits/32356ea1b9ddbf0c58eb34b67f0ef0f79035b859)